virtme_ng/run.py

```diff
@@ -523,6 +523,26 @@ def make_parser():
         + "or to launch this command instead of a prompt (--client).",
     )
 
+    g_remote.add_argument(
+        "--console",
+        action="store",
+        nargs="?",
+        type=int,
+        const=2222,
+        metavar="PORT",
+        help="Shortcut for `--server console --port PORT`"
+    )
+
+    g_remote.add_argument(
+        "--ssh",
+        action="store",
+        nargs="?",
+        type=int,
+        const=2222,
+        metavar="PORT",
+        help="Shortcut for `--server ssh --port PORT`"
+    )
```

Comment:
Why not dropping `--server` then?

Or have `--{console,ssh}-{client,server}` and drop `--port`?

`--client` and `--server` are maybe too generic?

Reply:
or `--{console,ssh}` for the server side (eventually with aliases to support `-server`), and ` --{console,ssh}-client` for the client part.

Reply:
Hm.. probably that's a better interface, I think we've already covered all the possible "servers" with console and ssh. And from a user perspective it seems a lot more convenient to just do `vng --console ...` or `vng --ssh ...` to start the server.

Reply:
Indeed, I understand the modification you did here. Then better not to confuse people with `--client` and `--server`. (then the README file needs to be adapted)